### ISSUE TITLE
Invalid indention on opening command {% not detected

### ISSUE THREAD

**Issue by c-po** [aristanetworks/j2lint#30]
```python
{% if foois defined%}
 {%     set bar= 'baz' %}
{% endif %}
```

The additional leading whitespace is not detected

**Comment by ClausHolbechArista** [comment]
This is valid jinja code, so the linter should not make any errors here. We have places in templates where we use `... {% include ... %}` so we cannot enforce that `{%` must be at the beginning of the line.
